src/main/resources/db/changelog/changes/create_reading_room_servicepoint.xml

```diff
@@ -19,4 +18,10 @@
     </createTable>
   </changeSet>
 
+  <changeSet id="addIndexesToReadingRoomServicePointTable" author="singhAdesh">
```

Comment:
Do we need to create a service point for primary key column?

Reply:
fixed